### PR TITLE
Define Singleton Admin Meta GraphQL API 

### PR DESCRIPTION
**Pull request by emmatown**
This defines the GraphQL types needed for singletons in the admin meta and updates the usages to continue working for lists. The implementation assumes singletons are never actually used. The Admin UI and the back-end portions of singletons can be properly implemented independently after this.

Based on #7845

### CODE REVIEW

File: .changeset/fair-terms-beg.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+'@keystone-6/core': patch
+'@keystone-6/fields-document': patch
+---
+
+Updates admin meta GraphQL schema to support singletons
```
Comment (flexdinesh) [inline .changeset/fair-terms-beg.md:6]: @mitchellhamilton will this be a separate patch to support singleton or will this go out together with singleton release as a major?

**Review by flexdinesh — APPROVED** [review]
Probably the docs change for `lists` => `models` will be up in some other PR yeah?

File: .changeset/fair-terms-beg.md (hunk 1 above)
Comment (mitchellhamilton) [reply .changeset/fair-terms-beg.md:6]: This will go out in the same release
